ee/tools/puppet/fleetdm/lib/puppet/reports/fleetdm.rb

```diff
@@ -8,15 +8,21 @@
 
   def process
     return if noop
-    client = Puppet::Util::FleetClient.instance
+
     node_name = Puppet[:node_name_value]
+    if resource_statuses.any? { |r| r.include?('error pre-setting fleetdm::profile') }
```

Comment:
Should we make the error string a constant so we don't forget to change it in multiple places if it gets updated in the future?

Reply:
good idea!! I don't know (not saying it's not possible) if we can easily do this since the error is raised in the `.pp` file.

I'll take a look if there's a good way to do it.

Reply:
didn't find a good way to do this, merging for now